docs/dapp/emerald/chain.mdx

```diff
@@ -0,0 +1,97 @@
+---
+description: Chain Information for Emerald
+---
+
+import {AddEmeraldToMetaMask as E, AddEmeraldTestnetToMetaMask as ET} from '@site/src/AddToMetaMask';
+
+# Chain Information
```

Comment:
```suggestion
# Network Information
```

was suggested by @rube-de

Reply:
I would also rename the file to network.mdx